sql/updates/world/4.3.4/2021_07_23_00_world.sql

```diff
@@ -0,0 +1,360 @@
+SET @CGUID := 399000;
+INSERT INTO `creature` VALUES (@CGUID, 44632, 0, 130, 5369, 1, 0, 1, 169, 0, -1, 0, 0, 1360.47, 1063.27, 52.9999, 3.92421, 300, 0, 0, 1, 0, 2, 0, 0, 0, '', 0);
+INSERT INTO `creature` VALUES (@CGUID+1, 44632, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1428.24, 1032.64, 53.0474, 4.06429, 300, 0, 0, 1398, 0, 2, 0, 0, 0, '', 0);
+
+INSERT INTO `creature_addon` VALUES (@CGUID, 446320, 0, 0, 0, 2, 0, 0, 0, 0, 0, '');
+INSERT INTO `creature_addon` VALUES (@CGUID+1, 446321, 0, 0, 0, 2, 0, 0, 0, 0, 0, '');
+
+ -- High Warlord Cromush
+INSERT INTO `creature` VALUES (@CGUID+2, 44640, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1357.21, 1046.36, 52.7554, 4.46892, 300, 0, 0, 387450, 0, 2, 0, 0, 0, 'npc_high_warlord_cromush_44640', 0);
+
+INSERT INTO `creature_addon` VALUES (@CGUID+2, 446400, 0, 0, 0, 1, 0, 0, 0, 0, 3, '80697');
+
+ -- Worg
+SET @ENTRY := 1765;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Ferocious Grizzled Bear
+SET @ENTRY := 1778;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Giant Rabid Bear
+SET @ENTRY := 1797;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Rabid Dog
+SET @ENTRY := 1766;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Vile Fin Oracle
+SET @ENTRY := 1908;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Vile Fin Tidehunter
+SET @ENTRY := 1768;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Skitterweb Lurker
+SET @ENTRY := 1781;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Skitterweb Lurker
+SET @ENTRY := 1781;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Bloodfang Scavenger
+SET @ENTRY := 44547;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Bat Handler Maggotbreath
+INSERT INTO `creature` VALUES (@CGUID+3, 44825, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1423.08, 1016.46, 52.544, 2.90902, 300, 0, 0, 77490, 0, 0, 0, 0, 0, NULL, 0);
+
+ -- Deathstalker Commander Belmont
+INSERT INTO `creature` VALUES (@CGUID+4, 44789, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1415.07, 1010.24, 52.7315, 2.09687, 300, 0, 0, 2085, 0, 0, 0, 0, 0, NULL, 0);
+
+-- High Apothecary Shana T'veen
+INSERT INTO `creature` VALUES (@CGUID+5, 44784, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1431.55, 1086.89, 60.4768, 3.81387, 300, 0, 0, 276, 590, 0, 0, 0, 0, NULL, 0);
+
+REPLACE INTO `creature_equip_template` VALUES (44365, 1, 2179, 0, 42775, 15595);
```

Comment:
No. Delete from first and then insert. No replace into.

Reply:
Fixed.